haproxy/default_frontend.cfg

```diff
@@ -1,28 +1,12 @@
-# Setting `log` here with the address of 127.0.0.1 will have the effect
-# of haproxy sending the udp log messages to its own rsyslog instance
-# (which sits at `127.0.0.1`) at the `local0` facility including all
-# logs that have a priority greater or equal to the specified log level
-# log 127.0.0.1 local0 warning
 global
-  maxconn 150000
+  maxconn 60000
   spread-checks 5
   lua-load-per-thread /usr/local/etc/haproxy/parse_basic.lua
   lua-load-per-thread /usr/local/etc/haproxy/parse_cookie.lua
   lua-load-per-thread /usr/local/etc/haproxy/replace_password.lua
   log stdout len 65535 local2 debug
-  tune.bufsize 1638400
-  tune.http.maxhdr 1010
-
-# https://www.haproxy.com/documentation/hapee/latest/onepage/#3.2-tune.bufsize
-# At least the global maxconn
-# parameter should be decreased by the same factor as this one is increased. If an
-# HTTP request is larger than (tune.bufsize - tune.maxrewrite), HAProxy will
-# return HTTP 400 (Bad Request) error. Similarly if an HTTP response is larger
-# than this size, HAProxy will return HTTP 502 (Bad Gateway).
-
-# https://www.haproxy.com/documentation/hapee/latest/onepage/#3.2-tune.http.maxhdr
-# Similarly, too large responses
-# are blocked with "502 Bad Gateway".
+  tune.bufsize 32768
+  tune.buffers.limit 60000
```

Comment:
The [documentation](https://www.haproxy.com/documentation/hapee/2-4r1/onepage/#tune.buffers.limit) suggests a much lower number here. My interpretation is they think it should be about 4 so we're about 15000x over. Also, this:

> It is best not to touch this value unless advised to do so by an HAProxy core developer.

Why this number? Please add a code comment.

Reply:
From tests,  I'm seeing a lot more 502 errors with 1/10 tune.buffers.limit. 
I'll add a detailed statistic tomorrow.

Reply:
I couldn't get a single scalability workflow to work using `tune.buffers.limit 6000` :( I've reverted the change.

Reply:
Thanks for trying! Let's run with what we've got and revisit when we evaluate auditing solutions later...